bot/extensions/adventofcode/views.py

```diff
@@ -0,0 +1,117 @@
+import re
+from datetime import datetime
+from zoneinfo import ZoneInfo
+
+import discord
+from bs4 import BeautifulSoup
+from discord import ui
+
+from bot import core
+from bot.extensions.adventofcode.utils import LEADERBOARD_ID, YEAR, Member, fetch_leaderboard, home_embed, ordinal
+
+
+class CreateAdventOfCodeView(ui.View):
+    HOME_CUSTOM_ID = "extensions:adventofcode:home"
+    LOCAL_LEADERBOARD_CUSTOM_ID = "extensions:adventofcode:local"
+    GLOBAL_LEADERBOARD_CUSTOM_ID = "extensions:adventofcode:global"
+
+    def reset_buttons(self):
+        self.children[0].disabled = True
+        self.children[1].disabled = False
+        self.children[2].disabled = False
```

Comment:
I don't think we need to disable the buttons, they can use them to reload the page theire on